src/lib/babel-custom.js

```diff
@@ -141,6 +138,7 @@ export default () => {
 									},
 									preset.options,
 									{
+										bugfixes: customOptions.modern,
```

Comment:
This can just always be set to `true`, it has no effect on cases where the output is below the affected syntax level.

Reply:
Sweet